Ch integrate houndci 160451589

obulaworld opened the pull request:
#### What does this PR do?
-Integrate HoundCI to the project

#### Description of Task to be completed?
- Integration of houndci to the project to also check for conventions.
- Attached HoundCI badge to the readme file.

#### How should this be manually tested?
- Does not need testing.

#### What are the relevant pivotal tracker stories?
[Integrate HoundCI with badge attached to readme](https://www.pivotaltracker.com/story/show/60451589)

Commit:
chore(eslint): integrate eslint

Integrate Eslint to the project
to make sure the team is following coding conventions

closes 160451588

Commit:
chore(houndci): integrate houndci

Integrate HoundCI to the project
to also make sure the team is following coding conventions

closes 160451589

Mrshinezee reviewed (approved):
LGTM

Victor-Ugwueze reviewed (approved):
LGTM

KaiserPhemi commented on .eslintrc.js:
```diff
@@ -0,0 +1,33 @@
+          }
+      }]
+    }
+};
```
#### Nitpick
Insert new line at the end of file

obulaworld replied:
Added the new line. Thanks for the feedback

KaiserPhemi commented on .hound.yml:
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
#### Nitpick
Insert new line at the end of file

KaiserPhemi commented:
#### Critical
@obulaworld 
PT user story link attached in the PR message is broken. Please check the link again.

Commit:
chore(eslint): integrate eslint

Integrate Eslint to the project
to make sure the team is following coding conventions (added new line)

closes 160451588

KaiserPhemi commented on README.md:
```diff
@@ -1,3 +1,4 @@
+[![protected_by hound](https://img.shields.io/badge/protected%20by-Hound-blue.svg)](https://houndci.com/repos)
 Authors Haven - A Social platform for the creative at heart.
```
#### Nitpick
Can you please insert a line between the badge and the title. This is still not properly formatted.

<img width="977" alt="screen shot 2018-09-25 at 3 11 03 pm" src="https://user-images.githubusercontent.com/26272628/46020046-574c8600-c0d5-11e8-916f-29941868654a.png">

obulaworld replied:
Okay...on it right away

obulaworld replied:
Just added the new line to the readme file. Thanks for the feedback

Commit:
chore(houndci): integrate houndci

Integrate HoundCI to the project
to also make sure the team is following coding conventions (ammended)

closes 160451589
